app/views/organisations/not_live.html.erb

```diff
@@ -22,7 +22,7 @@
     <div class="inner-block">
       <div class="description">
         <%= govspeak_to_html @organisation.summary %>
-        <% unless @organisation.closed? %>
+        <% unless @organisation.closed? || @organisation.court_or_hmcts_tribunal? %>
```

Comment:
To clarify, this is not going to be the case for live courts, as this is in the `not_live` template.